src/app/view/IAP/ClouderyOffer.tsx

```diff
@@ -68,6 +88,29 @@ const WebViewWithLoadingOverlay = ({
   )
 }
 
+interface BackButtonProps {
+  onPress: () => void
+}
+
+const BackButton = ({ onPress }: BackButtonProps): JSX.Element => {
+  const { t } = useI18n()
+
+  return (
+    <View style={styles.headerStyle}>
+      <TouchableOpacity
+        activeOpacity={0.5}
+        onPress={onPress}
+        style={styles.headerTouchableStyle}
+      >
+        <BackTo color={colors.primaryColor} width={16} height={16} />
+        <Text style={styles.headerTextStyle}>
+          {t('screens.clouderyOffer.backButton')}
+        </Text>
+      </TouchableOpacity>
+    </View>
+  )
+}
+
 const styles = StyleSheet.create({
```

Comment:
starting to have a lot of styles LoC in that file maybe we can extract it. Also it seems some of the value are magic numbers at first glance